Gemfile

```diff
@@ -8,6 +8,8 @@ source 'https://rubygems.org'
 # favorite editor. See https://github.com/firebase/firebase-ios-sdk/pull/8498
 # for examples.
 
+# TODO(ncooke3): Revert!
```

Comment:
revert

Reply:
Done in [7920b09](https://github.com/firebase/firebase-ios-sdk/pull/9661/commits/7920b09463826c7a822bc0370aa0927bef00ee06)